setup.py

```diff
@@ -39,7 +39,7 @@
 
 setup(
     name = 'CouchDB',
-    version = '1.2.1',
+    version = '1.2.2-m3d',
```

Comment:
This is not a canonical version and might cause issues. See https://peps.python.org/pep-0440/

I would go for `'1.2.1.post1+m3d'`
or add an epoch `'1!1.2.1+m3d'`